mlflow/store/artifact/azure_data_lake_artifact_repo.py

```diff
@@ -59,38 +74,34 @@ class AzureDataLakeArtifactRepository(ArtifactRepository):
     def __init__(self, artifact_uri, credential):
         super().__init__(artifact_uri)
         _DEFAULT_TIMEOUT = 600  # 10 minutes
+        self.credential = credential
         self.write_timeout = MLFLOW_ARTIFACT_UPLOAD_DOWNLOAD_TIMEOUT.get() or _DEFAULT_TIMEOUT
 
         (filesystem, account_name, path) = _parse_abfss_uri(artifact_uri)
 
+        # TODO: investigate setting the account URL based on whether the abfss URI is associated
+        # with an Azure account in standard Azure, govcloud, mooncake, etc
```

Comment:
Punting this investigation into future work

Reply:
`get_databricks_env_vars` has this comment:

```
    # We set these via environment variables so that only the current profile is exposed, rather
    # than all profiles in ~/.databrickscfg; maybe better would be to mount the necessary
    # part of ~/.databrickscfg into the container
```

@dbczumar Any risk if we use all profiles in ~/.databrickscfg?